docs/source/guides/session_limits.rst

```diff
@@ -0,0 +1,75 @@
+How To Set Orchestrator Session Limits
+======================================
+
+What Are Orchestrator Sessions?
+******************************
+
+By default an Orchestrator node can handle a maximum of **10** incoming livestreams at a time. Whenever the number of concurrent transcoding sessions go above this, the Orchestrator returns an ``OrchestratorCapped`` error to the Broadcaster node.
+
+Of course, the transcoding capacity and the bandwidth an Orchestrator has varies for everyone - so you can set the ``-maxSessions`` cli parameter with the correct number to fully utilize their hardware and get maximum work from the network.
+
+For example if an Orchestrator has enough resources (i.e. GPUs/CPUs/Bandwidth) to handle ``30`` streams instead you can pass it while starting up the node -
+
+::
+
+    $ livepeer -orchestrator -transcoder ... -maxSessions 30
```

Comment:
Since it is quite common for readers to just copy and paste commands from guides such as this one, I suggest removing the "..." and adding a note that ends up looking like this:

"to handle 30 streams you can set its max sessions using the following command (other flags omitted):"